crypto/x509/x509_req.c

```diff
@@ -286,6 +286,15 @@ void X509_REQ_get0_signature(const X509_REQ *req, const ASN1_BIT_STRING **psig,
         *palg = &req->sig_alg;
 }
 
+void X509_REQ_set0_signature(X509_REQ *req, ASN1_BIT_STRING *psigOrNull,
+                             X509_ALGOR *palgOrNull)
```

Comment:
Correction: I just noticed that it does not make sense to call the third parameter `sig_alg` because it is dereferenced. 

```
        req->sig_alg = *sig_alg;
```
But then at least I would call it `palg` (as in the getter) or `psig_alg`.

https://github.com/openssl/openssl/blob/429f4997de63209da2a90cdcf617587dab338182/crypto/x509/x509_req.c#L280-L296

Reply:
_Side note: The getter shows nicely how useless hungarian notation is, in particular if it is not used correctly: it should be `**ppalg`, not `**palg` ;-)_

Reply:
Thanks - should be fixed in the updated PR.